include/actuator/ActuatorTypes.h

```diff
@@ -16,15 +16,14 @@
 namespace sierra {
 namespace nalu {
 
-#ifdef ACTUATOR_ON_DEVICE
+#ifdef KOKKOS_ENABLED_CUDA
```

Comment:
I think there is a typo here. It is `KOKKOS_ENABLE_CUDA` not `ENABLED`

Reply:
ah thanks